cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
```

Comment:
Ensure that the `attrs` dictionary contains the keys 'row', 'seat', and 'movie_session' before accessing them. You might want to add checks or handle potential `KeyError` exceptions to prevent runtime errors if these keys are missing.